package.json

```diff
@@ -197,6 +197,7 @@
     ]
   },
   "resolutions": {
+    "fp-ts": "^2.11.1",
```

Comment:
why this resolution?

Reply:
Because `fp-ts` version in `io-functions-commons` is slightly different and is broking the build.